toolchain.mk

```diff
@@ -61,7 +81,27 @@ endef
 clang-toolchains:
 	$(call dl-clang,$(CLANG_VER),$(CLANG_PATH))
 
-else # $(UNAME_M) != x86_64
+else ifeq ($(UNAME_M),aarch64)
+
+AARCH32_PATH 			?= $(TOOLCHAIN_ROOT)/aarch32
+AARCH32_CROSS_COMPILE 		?= $(AARCH32_PATH)/bin/arm-linux-gnueabihf-
+AARCH32_GCC_VERSION 		?= gcc-arm-10.2-2020.11-aarch64-arm-none-linux-gnueabihf
+SRC_AARCH32_GCC 		?= https://developer.arm.com/-/media/Files/downloads/gnu-a/10.2-2020.11/binrel/$(AARCH32_GCC_VERSION).tar.xz
+
+AARCH64_PATH 			?= $(TOOLCHAIN_ROOT)/aarch64
+AARCH64_CROSS_COMPILE 		?= $(AARCH64_PATH)/bin/aarch64-linux-
```

Comment:
Why not use the native compiler directly? (by native I mean `/usr/bin/gcc` or whatever -- the compiler that comes with the Linux distribution).

Reply:
That's the first thing I tried and it worked for standalone components but failed while building buildroot as it doesn't supports build using native toolchain (refer [here](https://buildroot.org/downloads/manual/manual.html#_cross_compilation_toolchain)). And then I tried to fake cross-compiler for buildroot as native compiler but that too failed while preparing sysroots and runtime libraries for rootfs.

Reply:
Alright thanks for these details. Could you mention that buildroot doesn't support the native toolchain (with the link) as a comment here perhaps?

Reply:
Updated commit description accordingly.

Reply:
Added comment in the code as well.